### PR TITLE
Update export-teams-content.md

### PR DESCRIPTION
**Pull request by Vikramju**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/export-teams-content.md [hunk 1]
```diff
@@ -69,8 +69,9 @@ Here are some examples on how you can use these export APIs:
 - Microsoft Teams APIs in Microsoft Graph that access sensitive data are considered protected APIs. Export APIs require that you have additional validation, beyond permissions and consent, before you can use them. To request access to these protected APIs, complete the [request form](https://aka.ms/teamsgraph/requestaccess).
 - Application permissions are used by apps that run without a signed-in user present; application permissions can only be consented by an administrator. The following permissions are needed:
 
-    - *Chat.Read.All*: enables access to all 1:1 and Group chat messages 
-    - *User.Read.All*: enables access to the list of users for a tenant 
+    - *Chat.Read.All*: enables access to all 1:1, Group chat and meeting chat messages
```
Comment (JohanFreelancer9) [inline Teams/export-teams-content.md:72]:
```suggestion
    - *Chat.Read.All*: enables access to all 1:1, Group chat, and meeting chat messages 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JohanFreelancer9** [comment]
@Vikramju - please commit my suggestion. Thanks.

**Pull request merged**